File: Dart/terminal.dart
import "dart:core";
import "dart:io";

void main() {
  // User input
  userInput();
  // User output
  writeToTerminal();
  // Clear the terminal
  clearTerminal();
}

// Take in user input and print it back to the user
void userInput() {
  print("Enter your name");
  final String? name = stdin.readLineSync();
  print("Hello, ${name}");
}

// Write some text to the terminal
void writeToTerminal() {
  stdout.writeln("Hello, World");
}

// Clear the terminal
void clearTerminal() {
  stdout.write("\x1B[2J\x1B[0;0H");
}

File: C/terminal.c
#include <stdio.h>

// Clear the terminal screen
void clearTerminal() {
    printf("\033c");
}

// Take input from the user
int getUserInput() {
    int userInput;
    printf("Enter an integer:");
    scanf_s("%d", &userInput);
    return userInput;
}

int main() {
    // Clear the terminal
    clearTerminal();
    // Take input from the user
    //int userInput = getUserInput();
    //printf(userInput);
    return 0;
}
